modules/core/shared/src/main/scala/lucuma/core/model/Target.scala

```diff
@@ -24,18 +40,94 @@ object Target {
       Order.by(_.toInt)
   }
 
+  implicit val TargetEq: Eq[Target] =
+    Eq.by(x => (x.name, x.catalogId, x.track, x.magnitudes))
+
   /** A target order based on tracking information.  For sidereal targets this
     * roughly means by base coordinate without applying proper motion.  For
     * non-sidereal this means by `EphemerisKey`.
+    *
+    * Not implicit.
     */
-  implicit val TargetTrackOrder: Order[Target] =
-    Order.by(t => (t.track, t.name))
+  val TargetTrackOrder: Order[Target] =
```

Comment:
Is this fallout from having multiple `Order` implicits for the band?

Reply:
It doesn't seem possible to define an order for `SortedMap` (which is kind of ironic), thus the `implicit Order` wouldn't be consistent with `Eq`. I preferred then to make it non implicit and create a separate `Eq`

Reply:
I'm not sure I understand.  `Eq` is implied by `Order`.